### DISCUSSION TITLE
Netflix

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#58] [General]
# Netflix

Check if Netflix is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/e736cdd2-60d5-4ba1-8ba2-63e0c9c8b89e

**Comment by ppavle** [comment]
This is not native app. This is just a simple Edge app.

**Reply by OpenSource03** [reply]
How so? https://apps.microsoft.com/detail/9wzdncrfj3tj?hl=en-US&gl=US seems like a real app to me...

**Reply by ppavle** [reply]
Just install and try it. Lot of new "apps" are like that. for example, disney+. 
You can actually "create an app" out of any website like that by yourself. Open website in Edge, click on "...", Apps, Install this site as an app. That's it.

**Reply by OpenSource03** [reply]
I see. Well, while it is a web app, it's still an app in the store and users may want to check it's status. Most of them have no idea this is just an edge app, just like I didn't know. That being said, Edge and Windows browser components are ARM native, so we will keep this app as ARM Native :)

**Reply by ppavle** [reply]
Makes sense. If it looks like an app, works like an app, installs like an app, it probably is an app :)

**Comment by scottt0** [comment]
Yes, Netflix is compatible with Windows ARM devices, including those powered by the Snapdragon X Elite processor. Native support for Netflix and other popular apps like Prime Video and Spotify ensures that ARM-based Windows devices, such as the Surface models, provide a smooth multimedia experience without relying on emulation. Windows ARM has been significantly enhanced with app compatibility, making it a strong choice for users looking for power-efficient and versatile devices​
TECHRADAR
​
WINDOWS CENTRAL
.

For detailed reviews of ARM-compatible apps or alternative solutions, you can explore platforms like [Netflix Mod APK](https://nitflixapk.com/), which provides tailored experiences for various devices.
